app/com/linkedin/drelephant/spark/heuristics/ConfigurationHeuristic.scala

```diff
@@ -77,6 +74,14 @@ class ConfigurationHeuristic(private val heuristicConfigurationData: HeuristicCo
       new HeuristicResultDetails(
```

Comment:
Sorry, missed your reply. An example would be 

   if( serializer check fails) 
     HeuristicResult.add( new HeuristicResultDetails(
         "Serializer",
         "KyroSerializer Not Enabled.") )
 if( dynamicallocation check fails) {
   HeuristicResult.add (
   new HeuristicResultDetails(
         "DynamicAllocation without Shuffle",
         "DynamicAllocation is enabled but."))
}